xsuite-lightsimulnet/src/handleAdmin.go

```diff
@@ -54,6 +54,24 @@ func (e *Executor) HandleAdminSetAccounts(r *http.Request) (interface{}, error)
 	return jData, err
 }
 
+func (e *Executor) HandleAdminUpdateAccounts(r *http.Request) (interface{}, error) {
+	reqBody, _ := io.ReadAll(r.Body)
+	var rawAccounts []UpdateRawAccount
+	err := json.Unmarshal(reqBody, &rawAccounts)
+	if err != nil {
+		return nil, err
+	}
+	for _, rawAccount := range rawAccounts {
+		err = e.updateAccount(rawAccount)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return map[string]interface{}{
```

Comment:
Let's introduce the variable `jData`.

In `HandleAdminSetAccounts`, let's return `return jData, nil`, which is indeed better.